package.json

```diff
@@ -107,6 +107,7 @@
       "!src/shared/*",
       "!src/graphql/*",
       "!src/context/*",
+      "!src/hooks/*",
```

Comment:
We should add unit tests to this folder, also to `context` folder

Reply:
Fixed here [#90d27e2](https://github.com/BuildIt-Poland/buddy-app/pull/61/commits/90d27e2ca5c70e562b45c061adbf45419e4df24f)